<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>616</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>952</width>
    <height>616</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>952</width>
    <height>616</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Стрелок</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>9</y>
      <width>941</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Список курсантов</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>911</width>
        <height>381</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>190</number>
      </attribute>
      <column>
       <property name="text">
        <string>ФИО</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>№ группы</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Оценка за 1 упражнение</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Оценка за 2 упражнение</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Оценка за 3 упражнение</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Итоговая оценка</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>911</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Поиск записи</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>314</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер учебной группы:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Поиск</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Сдаваемые упражнения</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>911</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Список упражнений</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>891</width>
         <height>171</height>
        </rect>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>250</number>
       </attribute>
       <column>
        <property name="text">
         <string>Номер упражнения</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Описание упражнения</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>911</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Нормативы упражнений</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>891</width>
         <height>221</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>180</number>
       </attribute>
       <column>
        <property name="text">
         <string>Номер упражнения</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Норматив 1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Оценка 1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Норматив 2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Оценка 2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Норматив 3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Оценка 3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Норматив 4</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Оценка 4</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Добавление записи</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>320</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Вывести в Word</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>551</width>
        <height>471</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <column>
       <property name="text">
        <string>ФИО</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Номер учебной группы</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Название 1 упражнения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Оценка за 1 упражнение</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Название 2 упражнения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Оценка за 2 упражнение</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Название 3 упражнения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Оценка за 3 упражнение</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Итоговая оценка</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>0</y>
        <width>341</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Добавить запись:</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ФИО:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>271</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Иванов И.И.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер учебной группы:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>314</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер упражнения 1:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Оценка за упражнение :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>120</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>270</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>180</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Оценка за упражнение :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер упражнения 2:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>240</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Оценка за упражнение :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер упражнения 3:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>210</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>370</y>
        <width>341</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Удаление записи:</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Выбранная строка:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>911</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Подключение к серверу:</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IP-адрес::</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>5555</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Порт:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Подключиться</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Создать файл БД</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Открыть файл БД</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_4">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Закрыть программу</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
